app/assets/javascripts/tabs.js

```diff
@@ -24,36 +24,45 @@ $( document ).ready(function() {
       break;
     default:
       $( "#component-summary" ).addClass( "active" );
+      $( "#summary-tab" ).addClass( "active" );
   }
 
-  $(".lux-tabs-container li").click(function( event ) {
-    event.preventDefault();
-    $(".lux-tabs-container li").removeClass( "active" );
-    $(".tab-pane").removeClass( "active" );
-    $("#toc .jstree-clicked").removeClass( "jstree-clicked" );
-    switch (event.currentTarget.id) {
-      case 'summary-tab':
-        $( "#summary-tab" ).addClass( "active" );
-        $( "#summary" ).addClass( "active" );
-        break;
-      case 'description-tab':
-        $( "#description-tab" ).addClass( "active" );
-        $( "#description" ).addClass( "active" );
-        break;
-      case 'collection-history-tab':
-        $( "#collection-history-tab" ).addClass( "active" );
-        $( "#collection-history" ).addClass( "active" );
-        break;
-      case 'access-tab':
-        $( "#access-tab" ).addClass( "active" );
-        $( "#access" ).addClass( "active" );
-        break;
-      case 'find-more-tab':
-        $( "#find-more-tab" ).addClass( "active" );
-        $( "#find-more" ).addClass( "active" );
-        break;
-      default:
-        $( "#summary-tab" ).addClass( "active" );
-    }
+  $(".lux-tabs-container li").click(function() {
+    let temp_id = $(this).attr('id').split("-");
+    temp_id.pop()
+    window.location.hash = temp_id.join("-");
+    console.log($(this).attr('id'))
+    window.location.reload()
   });
+
+  // $(".lux-tabs-container li").click(function( event ) {
```

Comment:
Can you delete this instead of commenting?

Reply:
Yes. I added a permalink to the old code for future reference.